src/App.js

```diff
@@ -52,16 +53,38 @@ class PanelButtons extends Component {
     }
   };
 
-  localizedSaveMessage = (saveStatus) => {
-    return I18n.t(`saveStatus_${saveStatus}`)
+  localizedSaveMessage = saveStatus => {
+    return I18n.t(`saveStatus_${saveStatus}`);
+  };
+
+  saveResponseDataMessage = saveResponseData => {
+    // The list of known error types from share_filtering.rb.
+    const errorTypes = ["email", "address", "phone", "profanity"];
```

Comment:
It would be handy to have a single source of truth for these, and we do solve that in the main repo by having some constants in JS that are generated from Ruby.  However, I'm not sure of a great automated way to get this from the main repo across to this standalone one.